.gitignore

```diff
@@ -47,6 +47,7 @@ phpcs.xml
 ## Things we will need in release branches
 /_inc/blocks
 /_inc/build
+/_inc/search/dist/
```

Comment:
That's an helpful addition, thank you. 

We would also need to add the files we don't want to ship in a release in `.svnignore`:
https://github.com/Automattic/jetpack/blob/ee0fcec521f94cb2596ea4a8879350ca098457ae/.svnignore#L52

Could you speak a bit about your choice of a new subdirectory of `_inc/search` for the built files? I wonder if it may be easier to decide what to ship if the built files were in `_inc/build` instead?

Reply:
The entire `_inc/search` directory is new. I only added `dist` because that is how the existing code was structured. I don't think there is any reason not to move them into `_inc/build` especially if that helps us hook into the existing Jetpack JS CDN in any way (do you know how that works?).

Is moving them your recommendation? I was also concerned that the build dir may be mostly for the dashboard client build.

Reply:
In longer-term it could be helpful to get everything under `_inc/build` (including blocks), using one Webpack config file just with several different entries.

> I was also concerned that the build dir may be mostly for the dashboard client build.

I think that is mostly true but only because the admin dash has been the only bit of ES6 code in Jetpack until recently. It can be more clear to think the separation of code in terms of ES6 vs ES5 instead of "admin dash vs other features". 

If it's easy to move files to `_inc/build`, I'd go ahead and do that and not set too strong precedence of having different sub-directories for each feature.

Blocks are now built with the separate config to a separate folder because we couldn't untangle how building admin dash works and combine the two pipelines into one (https://github.com/Automattic/jetpack/issues/11745). Some work has since happened to make it easier tho.

Block extensions also have a mechanism to build those files for .com on Phabricator commits, so piggybacking on that pipeline could be useful to you.

Reply:
> I don't think there is any reason not to move them into _inc/build especially if that helps us hook into the existing Jetpack JS CDN in any way (do you know how that works?).

You done have to worry about Jetpack's file CDN; as long as the files are in SVN, they'll be available and loaded via the CDN when the module is on, regardless of their location in the plugin structure.

Reply:
> Is moving them your recommendation?

Yeah, I think it would be okay to move `_inc/search/dist` to `_inc/build`, alongside the other built files, where it will just be shipped with the other files with no additional changes needed.

Reply:
Great, thanks for the feedback. I've added moving the built files to `_inc/build` to that open issue, so we'll get around to it.